src/main/java/com/github/technus/tectech/compatibility/dreamcraft/DreamCraftRecipeLoader.java

```diff
@@ -2954,58 +2958,69 @@ public void run(EMTransformationRegistry transformationInfo) {
-                    GT_OreDictUnificator.get(OrePrefixes.plate, Materials.SpaceTime, 1L),
+                    GT_OreDictUnificator
+                            .get(OrePrefixes.plate, Materials.MagnetohydrodynamicallyConstrainedStarMatter, 1L),
                     (int) TierEU.RECIPE_MAX,
                     32768,
-                    500_000_000,
+                    (int) TierEU.RECIPE_UXV,
                     64,
-                    new ItemStack[] { ItemList.Casing_Dim_Bridge.get(64),
-                            GT_OreDictUnificator.get(OrePrefixes.block, Materials.Infinity, 64L),
-                            GT_OreDictUnificator.get(OrePrefixes.block, Materials.Infinity, 64L),
-                            GT_OreDictUnificator.get(OrePrefixes.block, Materials.Infinity, 64L),
-                            GT_OreDictUnificator.get(OrePrefixes.block, Materials.SpaceTime, 16L),
+                    new ItemStack[] { CustomItemList.StabilisationFieldGeneratorTier8.get(64),
+                            ItemList.Casing_Dim_Bridge.get(64), GT_OreDictUnificator.get("blockShirabon", 64L),
+                            GT_OreDictUnificator.get("blockShirabon", 64L),
+                            GT_OreDictUnificator.get(OrePrefixes.block, Materials.SpaceTime, 64L),
                             GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.Quantum, 16L),
                             GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.SpaceTime, 8L),
-                            GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.SpaceTime, 8L),
+                            GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Universium, 8L),
                             GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Infinity, 8L),
-                            GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Infinity, 8L),
-                            ItemList.Sensor_UMV.get(16L), ItemList.Emitter_UMV.get(16L),
-                            getModItem("eternalsingularity", "eternal_singularity", 16L) },
+                            GT_OreDictUnificator.get("plateDenseShirabon", 8L), ItemList.Sensor_UXV.get(16L),
+                            ItemList.Emitter_UXV.get(16L), getModItem("eternalsingularity", "eternal_singularity", 16L),
+                            Materials.Universium.getNanite(16), Materials.BlackDwarfMatter.getNanite(16),
+                            Materials.WhiteDwarfMatter.getNanite(16), },
                     new FluidStack[] { Materials.Neutronium.getMolten(32_768_000L),
                             Materials.SpaceTime.getMolten(4 * 36864L),
                             Materials.SuperconductorUMVBase.getMolten(4 * 36864L),
                             Materials.ExcitedDTEC.getFluid(4 * 36864L) },
                     getItemContainer("StargateShieldingFoil").get(1L),
-                    72_000,
-                    500_000_000);
+                    500_000 * 20,
```

Comment:
You can't just put an absurd amount of time for one single plate, when overclocking with AAL is much harder than normal asslines and when the EU production can't keep up for such OCs.

If you want the stargate uncraftable again it's ok, just yeet the recipes 😂 

500 000 seconds is just 138h 53 min 20s. It's totally absurd at UMV voltage. You will be able to do such thing when EU powergen will be fixed to match such stupid recipes via overclocks.

Reply:
same goes for lines 3022 and 3057

Reply:
Ok so after reflection I have adjusted this value down to 125k seconds, this is reasonable imo, if you gave AAL 6 OCs that is 1953s/part. In total this means all recipes will take 27 days in 1 AAL and cost ~50 quintillion EU. This seems eminently reasonable now that the EOH can in theory make trillions of EU/t. I would also remind people this is 1 AAL with 0 extra parallel, I think it is not unreasonable to expect people to have more than 1 parallel when making the absolute most memey item in one of the hardest modpacks ever made as a final goal.

These recipes will inevitably get adjusted again once people reach the actual content and start making it. I'm fully open to feedback but I do not see an issue with these EU/time numbers as they stand.